lib/index.js

```diff
@@ -12,6 +12,7 @@ const gatewayModes = Object.freeze({
         itdProdURL: "https://itd.zoop.one",
         itdStagingURL: "https://itd-staging.zoop.one",
         eSignV4URL: "https://esign.zoop.one",
+        digilockerV1URL: "https://gateway.zoop.one",
```

Comment:
/digilocker/v1 should be part of v1 base URL itself

Reply:
@thefoodiecoder It is added at line 361.

Reply:
What I meant was that do not add that there but keep it in the base URL itself. So for v2 URL will be 

`digilockerV2URL: "https://gateway.zoop.one/digilocker/v1/"`

Reply:
This was you need to change URL at one place only.

Reply:
@thefoodiecoder that URL should be a origin URL because we check it later when we receive the event form the other tabs on line 413.
<img width="570" alt="image" src="https://user-images.githubusercontent.com/17024120/151518922-2efabf74-92be-4337-981c-b4b9f3246d91.png">

Reply:
To fix that we can do one `startsWith()` check instead. What do you think?

Reply:
@thefoodiecoder Requested change is done.